bin/phpcs-audit.sh

```diff
@@ -17,14 +17,12 @@ file_path=${1:-$default_path}
 default_file_affix=""
 file_affix=${2:-$default_file_affix}
 
-ignore="\.idea,\.saas-cache,node_modules,vendor,bin,tests"
-
 declare -a types=("full" "source")
 
 echo Running PHPCS reports
 
 for type in ${types[@]}
 do
   echo Running PHPCS report: ${type}
-  ./vendor/bin/phpcs -v -s -p -d memory_limit="256M" --extensions="php" --report="${type}" --ignore="${ignore}" --report-file="report-${type}${file_affix}.txt" ${file_path}
+  ./vendor/bin/phpcs -v -s -p -d memory_limit="256M" --extensions="php" --report="${type}" --report-file="report-${type}${file_affix}.txt" ${file_path}
```

Comment:
I'd advise that as many of these args as possible, get moved into the `phpcs.xml.dist`. See https://github.com/GaryJones/plugin-boilerplate/blob/bfe875179f641797576f6cbcd6b45a7e3931c186/phpcs.xml.dist#L11-L17 as an example. It would usually just be possible to call `./vendor/bin/phpcs`, whether manually, from Composer script or a bash script, and have it run consistently.